images-list

```diff
@@ -303,6 +307,8 @@ k8s.gcr.io/sig-storage/livenessprobe rancher/mirrored-sig-storage-livenessprobe
 kiwigrid/k8s-sidecar rancher/kiwigrid-k8s-sidecar 0.1.151
 kiwigrid/k8s-sidecar rancher/kiwigrid-k8s-sidecar 1.1.0
 kiwigrid/k8s-sidecar rancher/mirrored-kiwigrid-k8s-sidecar 0.1.151
+lachlanevenson/k8s-kubectl rancher/mirrored-lachlanevenson-k8s-kubectl v1.22.2
```

Comment:
Why are we using this? Have we considered using `rancher/shell` or `rancher/kubectl` instead?

Reply:
@aiyengar2 :arrow_up:

Reply:
@aiyengar2 May I have an answer here ?

Reply:
Resolved per meeting. Commit b9f1a77abe6815fea9683a69fb8125c8aaf15f01